src/levels.jl

```diff
@@ -1,3 +1,13 @@
+# * Level
+
+"""
+    Level(csf, J)
+
+Given a [`CSF`](@ref) with a final [`Term`](@ref), a `Level`
+additionally specifies a total angular momentum ``J``. By further
+specifying a projection quantum number ``M_J``, we get a
+[`State`](@ref).
```

Comment:
Hmm. So the CSF here would have to be in LS coupling then?

Reply:
No, not true; in the `jj`-coupled case, there is however trivially only one `Level`, with the same value of `J` as the CSF.

Reply:
See https://github.com/JuliaAtoms/AtomicLevels.jl/pull/91/files/3596028a910bd6648aa32e3247b5a31bb1801125#diff-800c4b457b4d08bf3b3b3abaf4d29a4fe9e83ede3e72a22397948085e9ab7c74R80

Reply:
Ah, true that. While trivial, nothing stopping you from constructing a `Level` object in the `jj` case.